### PR TITLE
Fix mapping for internal TF for CentOS-Stream

### PR DESCRIPTION
**Pull request by kokesak**
In Convert2RHEL we want to start testing on CentOS-Stream-8. The current mapping returns RHEL-8.5 Nightly images. This PR fixes that.

### CODE REVIEW

**Comment by lachmanfrantisek** [comment]
@kokesak Thanks for sending the PR!

@thrix Can you please look at this to see if this makes sense from your point of view? (For me, it makes sense to be explicit and users can still use RHEL if set explicitly.)

**Comment by bocekm** [comment]
Some context for this change - in convert2rhel we've revived our efforts to allow conversion from CentOS Stream 8 to RHEL 8 (https://github.com/oamg/convert2rhel/pull/267). For conversions in general we need the internal Red Hat network available, so with the `use_internal_tf` set to `True` we need to have a way to run the tests on a CentOS Stream 8 machine. However with the current mapping, we'd get the `RHEL-8.5.0-Nightly` instead.

**Comment by lachmanfrantisek** [comment]
Small update:

We discussed this with Miro last week and the stream is currently not present in the internal range -- @bocekm, can you please ask for the Stream to be included? (I hope you've already made the request for something else so you might know better than me what needs to be done...)

**Comment by lachmanfrantisek** [comment]
Another update:

I get that wrongly. `CentOS-Stream-8` is [available in the internal composes(https://api.dev.testing-farm.io/v0.1/composes/redhat), `CentOS-Stream-9` is not there yet.

File: packit_service/worker/helpers/testing_farm.py [hunk 1]
```diff
@@ -362,8 +362,6 @@ def distro2compose(self, distro: str, arch: str) -> str:
                 return "Fedora-Rawhide-Nightly"
             if compose.startswith("Fedora-"):
                 return f"{compose}-Updated"
-            if compose == "CentOS-Stream-8":
-                return "RHEL-8.5.0-Nightly"
```
[the remark this answers is not in the document]
Comment (lachmanfrantisek) [reply packit_service/worker/helpers/testing_farm.py:366]: And can you please mention https://api.dev.testing-farm.io/v0.1/composes/redhat as well? (It was added just recently..) Thank you!

**Comment by bocekm** [comment]
@lachmanfrantisek, thanks for the update. We need just the `CentOS-Stream-8` for now and won't need 9 for many months ahead.

File: packit_service/worker/helpers/testing_farm.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (bocekm) [reply packit_service/worker/helpers/testing_farm.py:366]: Please give @kokesak a day or two. He should be back from vacation by tomorrow.

[the remark this answers is not in the document]
Comment (kokesak) [reply packit_service/worker/helpers/testing_farm.py:366]: I updated the docstring and added the https://api.dev.testing-farm.io/v0.1/composes/redhat to the comment above. Is that what you wnated @lachmanfrantisek ?

[the remark this answers is not in the document]
Comment (lachmanfrantisek) [reply packit_service/worker/helpers/testing_farm.py:366]: Thanks!

[the remark this answers is not in the document]
Comment (lachmanfrantisek) [reply packit_service/worker/helpers/testing_farm.py:366]: Oh. wait. I haven't checked the docstring as Jirka wanted.

Can you please remove this line:
```
* CentOS-Stream-8 -> RHEL-8.5.0-Nightly
```

[the remark this answers is not in the document]
Comment (kokesak) [reply packit_service/worker/helpers/testing_farm.py:366]: It's removed now.

**Review by lachmanfrantisek — APPROVED** [review]
Thanks!

**Pull request merged**